azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -158,6 +158,11 @@ void setup(final ExecutableFlow flow) throws ExecutorManagerException {
       FileIOUtils.deleteDirectorySilently(tempDir);
       LOGGER.error("Error in preparing flow execution {}", flow.getExecutionId(), ex);
       throw new ExecutorManagerException(ex);
+    } catch (final Throwable t) {
```

Comment:
If catching Throwable exceptions is not a temporary change please consider merging both catch blocks since they are doing almost the same.

Reply:
Yes agree with @ypadron-in. In fact, we can have just Throwable catch block if the intention is catch all types of Exception including RuntimeException.

Reply:
Thank you. It was temporary but I have added change to only catch throwable.